webviz_subsurface/plugins/_map_viewer_fmu/color_tables.py

```diff
@@ -0,0 +1,190 @@
+            [1.00, 74, 19, 86],
+        ],
+    },
+    {
+        "name": "Stratigraphy",
+        "discrete": True,
+        "colors": [
+            [0, 255, 120, 61],
+            [1, 255, 193, 0],
+            [2, 255, 155, 76],
+            [3, 255, 223, 161],
+            [4, 226, 44, 118],
+            [5, 255, 243, 53],
+            [6, 255, 212, 179],
+            [7, 255, 155, 23],
+            [8, 255, 246, 117],
+            [9, 255, 241, 0],
+            [10, 255, 211, 178],
+            [11, 255, 173, 128],
+            [12, 248, 152, 0],
+            [13, 154, 89, 24],
+            [14, 0, 138, 185],
+            [15, 82, 161, 40],
+            [16, 219, 228, 163],
+            [17, 0, 119, 64],
+            [18, 0, 110, 172],
+            [19, 116, 190, 230],
+            [20, 0, 155, 212],
+            [21, 0, 117, 190],
+            [22, 143, 40, 112],
+            [23, 220, 153, 190],
+            [24, 226, 44, 118],
+            [25, 126, 40, 111],
+            [26, 73, 69, 43],
+            [27, 203, 63, 42],
+            [28, 255, 198, 190],
+            [29, 135, 49, 45],
+            [30, 150, 136, 120],
+            [31, 198, 182, 175],
+            [32, 166, 154, 145],
+            [33, 191, 88, 22],
+            [34, 255, 212, 179],
+            [35, 251, 139, 105],
+            [36, 154, 89, 24],
+            [37, 186, 222, 200],
+            [38, 0, 124, 140],
+            [39, 87, 84, 83],
+        ],
+    },
+    {
+        "name": "Facies",
+        "discrete": True,
+        "colors": [
+            [0, 255, 193, 0],
+            [1, 255, 246, 117],
+            [2, 166, 194, 42],
+            [3, 149, 160, 24],
+            [4, 9, 143, 74],
+            [5, 125, 98, 15],
+            [6, 0, 108, 154],
+            [7, 0, 117, 190],
+            [8, 28, 22, 59],
+            [9, 39, 142, 199],
+            [10, 0, 138, 185],
+            [11, 52, 178, 188],
+            [12, 235, 63, 34],
+            [13, 74, 19, 86],
+            [14, 248, 152, 0],
+            [15, 1, 1, 1],
+            [16, 128, 128, 128],
+        ],
+    },
+    {
+        "name": "GasOilWater",
+        "discrete": True,
+        "colors": [
+            [0, 255, 46, 0],
+            [1, 0, 184, 0],
+            [2, 0, 25, 255],
+            [3, 179, 179, 179],
+        ],
+    },
+    {
+        "name": "GasWater",
+        "discrete": True,
+        "colors": [[0, 255, 46, 0], [1, 0, 25, 255], [2, 179, 179, 179]],
+    },
+    {
+        "name": "OilWater",
+        "discrete": True,
+        "colors": [[0, 0, 184, 0], [1, 0, 25, 255], [2, 179, 179, 179]],
+    },
+    {
+        "name": "Accent",
+        "discrete": True,
+        "colors": [
+            [0, 127, 201, 127],
+            [1, 190, 174, 212],
+            [2, 253, 192, 134],
+            [4, 255, 255, 153],
+            [5, 56, 108, 176],
+            [6, 240, 2, 127],
+            [7, 191, 91, 23],
+            [8, 102, 102, 102],
+        ],
+    },
+]

```

Comment:
Should we store this as JSON, and import with `json.load()`? Similar to how we have done it in https://github.com/equinor/webviz-subsurface/tree/master/webviz_subsurface/_abbreviations/abbreviation_data.